packages/sources/dar/src/transport/price.ts

```diff
@@ -80,4 +82,38 @@ export const transport = new WebSocketTransport<WsTransportTypes>({
       ]
     },
   },
-})
+}
+
+export class DarWebsocketTransport extends WebSocketTransport<WsTransportTypes> {
+  override async streamHandler(
+    context: EndpointContext<WsTransportTypes>,
+    subscriptions: SubscriptionDeltas<RequestParams>,
+  ): Promise<void> {
+    let subscribeAssets = ''
+    for (const pair of subscriptions.desired) {
+      subscribeAssets = subscribeAssets.concat(`${pair['base'].toString().toLowerCase()},`)
+    }
+
+    if (this.wsConnection && !this.connectionClosed() && subscriptions.new.length) {
```

Comment:
Should we also handle `stale` subscriptions? These are ones that have expired from the subscription set, and so we no longer need to fetch data for them.